itests/org.jboss.tools.openshift.ui.bot.test/README.md

```diff
@@ -3,20 +3,22 @@ This plugin contains OpenShift 3 tooling integrations tests. For execution of a
-Note: tested with CDK 3.1.1. with updated templates from https://github.com/minishift/minishift/pull/1435.
+Note: tested with CDK 3.2.0.alpha.
 
 #### Running OS tests from IDE
 There are three suites for OpenShift 3 tests (full suite (OpenShift3BotTests), smoke suite (OpenShift3SmokeBotTests)) and stable suite (OpenShift3StableTests). To run OpenShift tests from IDE, perform following steps:
 1. Install RedDeer to your IDE from update site: http://download.eclipse.org/reddeer/releases/latest (RedDeer is on DevStudio TP, RedDeer project repo URL: https://github.com/eclipse/reddeer)
 2. Select desired suite (e.g. OpenShift3BotTests, OpenShift3SmokeBotTests, OpenShift3StableBotTests) and in its context menu select _Run As_ - _Run Configurations..._
 3. In Run Configurations shell double click on RedDeer Test and a new RedDeer test run configuration for your suite is created
 4. Select tab Argument and fill in following properties **with credentials** to VM arguments:
-`-Dopenshift.server= -Dopenshift.username= -Dopenshift.password= -Dsecurestorage.password= -Dusage_reporting_enabled=false`
-for OpenShift 3 tests add also property `-Dopenshift.authmethod=basic`.   
+`-Dopenshift.server= -Dopenshift.username= -Dopenshift.password= -Dsecurestorage.password= -Dusage_reporting_enabled=false -Dopenshift.authmethod=basic`
+for OpenShift tests is used basic authentication method with username/password by default.   
 5. Confirm changes and run test suite.
 
 #### Running OS tests from command line
-For execution of OpenShift tests from command line you will need maven. At first build _org.jboss.tools.openshift.reddeer_ plugin with maven. Then run in _org.jboss.tools.openshift.ui.bot.test_ plugin following command **with filled credentials** with basic authentication. By default it is used OpenShift3StableBotTests tests suite. User can switch to differetn suite class by using profiles: `-Psmoke` (OpenShift3SmokeBotTests) or `-Pfull` (OpenShift3BotTests).
-`mvn clean verify -Dtest.installBase=/path/to/ide/to/run/against -Dopenshift.username= -Dopenshift.password= -Dsecurestorage.password= Dopenshift.authmethod=basic -Dopenshift.server= -DskipTests=false`
+For execution of OpenShift tests from command line you need to have installed maven. At first you need to build jbosstools-openshift repo with maven and disabled tests, e.g. `mvn clean install -DskipTests=true -DskipITests=true`. Then run in _org.jboss.tools.openshift.ui.bot.test_ plugin following command **with filled credentials and OpenShift server** with basic authentication. By default it is used OpenShift3StableBotTests tests suite. User can switch to different suite class by using profiles: `-Psmoke` (OpenShift3SmokeBotTests) or `-Pfull` (OpenShift3BotTests).
+`mvn clean verify -Dopenshift.username= -Dopenshift.password= -Dsecurestorage.password= -Dopenshift.server= `
```

Comment:
Running this command did not make any test to run. We must add either -DskipITests=false either -PITests

Reply:
thanks @jeffmaury , I have added -PITests in this command